### PR TITLE
Create resolve app

### PR DESCRIPTION
**Pull request by pglushenkov**

### CODE REVIEW

File: packages/create-resolve-app/src/create_resolve_app.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
+import spawn from 'cross-spawn';
```
Comment (IhostVlad) [inline packages/create-resolve-app/src/create_resolve_app.js]: Does this file appears to be major copy-paste from here: https://github.com/facebookincubator/create-react-app/blob/master/packages/create-react-app/createReactApp.js, with refactor to ES6 language ? 

Or source, such code is excepted solution, but what about licences? 

```
/**
 * Copyright (c) 2015-present, Facebook, Inc.
 * All rights reserved.
 *
 * This source code is licensed under the BSD-style license found in the
 * LICENSE file in the root directory of this source tree. An additional grant
 * of patent rights can be found in the PATENTS file in the same directory.
 */
```

If all ok, not problem

File: packages/resolve-scripts/src/template/README.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# Resolve boilerplate
```
Comment (IhostVlad) [inline packages/resolve-scripts/src/template/README.md]: Is it latest version of Boilerplate readme? Maybe perform merge latest version?

File: packages/create-resolve-app/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "create-resolve-app",
+  "version": "0.0.12",
+  "description": "This package generates boilerplate web application based on resolve framework",
+  "bin": {
+    "create-resolve-app": "./dist/bin/index.js"
```
Comment (IhostVlad) [inline packages/create-resolve-app/package.json]: Where is this file? If supposed `/src/index.js`, that it should contant shebang construction in first line

Comment (VadimKhoroshiltsev) [inline packages/create-resolve-app/package.json]: Yes, without this file it is impossible to install this package

**Pull request merged**